conf/tachyon-env.sh.template

```diff
@@ -76,7 +83,7 @@ export TACHYON_JAVA_OPTS+="
 "
 
 # Master specific parameters. Default to TACHYON_JAVA_OPTS.
-export TACHYON_MASTER_JAVA_OPTS="$TACHYON_JAVA_OPTS"
+export TACHYON_MASTER_JAVA_OPTS="$TACHYON_JAVA_OPTS -Dtachyon.master.maxworker.threads=$TACHYON_MASTER_MAX_THREADS"
 
 # Worker specific parameters that will be shared to all workers. Default to TACHYON_JAVA_OPTS.
-export TACHYON_WORKER_JAVA_OPTS="$TACHYON_JAVA_OPTS"
+export TACHYON_WORKER_JAVA_OPTS="$TACHYON_JAVA_OPTS -Dtachyon.worker.maxworker.threads=$TACHYON_WORKER_MAX_THREADS"
```

Comment:
The reason I did in this way is each of them only is used by either Master or Worker.